tests/installation/bootloader_ofw.pm

```diff
@@ -40,8 +40,15 @@ sub run() {
         || get_var('EXTRABOOTPARAMS'))
     {
         if (get_var("ZDUP") || get_var("ONLINE_MIGRATION") || get_var('PATCH')) {
-            send_key "down";
-            send_key "ret";
+            if (check_var('ARCH', 'ppc64le')) {
+                record_soft_failure 'bsc#1035428';
+                wait_screen_change { send_key 'ret' };
+                # specifi local disk for boot
+                type_string_very_slow "boot /pci\t/sc\t3";
```

Comment:
yes this is a magic number if it will not work, I will work on some crazy way ho to get correct number, but now I'm super fine with 3